bindings/nodejs/lib/utils/utils.ts

```diff
@@ -193,16 +194,18 @@ export class Utils {
     }
 
     /**
-     * Compute the block ID (Blake2b256 hash of the block bytes) of a block.
+     * Compute the block ID (Blake2b256 hash of the block bytes) of a blockwrapper.
      *
-     * @param block A block.
+     * @param wrapper A blockwrapper.
+     * @param params The network protocol parameters.
      * @returns The corresponding block ID.
      */
-    static blockId(block: Block): BlockId {
+    static blockId(wrapper: BlockWrapper, params: ProtocolParameters): BlockId {
```

Comment:
Honestly I wouldn't change the block/wrapper variables names. We haven't changed it in Rust as we're waiting for a conclusion for the naming discussion. I'd keep everything as it is except for type for now.